app/assets/stylesheets/main.scss

```diff
@@ -11,3 +14,7 @@
 
 // Pages CSS
 @import "pages/home";
+@import "pages/login";
+
+// Custom bootstrap variables must be set or imported *before* bootstrap.
+@import "bootstrap";
```

Comment:
bootstrap is already imported in line 6, so you should remove this line. Also ordering is important in CSS where specific css files (i.e files in `pages` folder`) comes **after** more general style imports (i.e. `bootstrap`)